diff --git a/CMakeLists.txt b/CMakeLists.txt
index d4e50b993..51a3a06d1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -84,7 +84,7 @@ include(FindPython)
 message(STATUS "Generating cmake files ...")
 execute_process(
   COMMAND ${PYTHON}
-    -m pip install --disable-pip-version-check
+    -m pip install --disable-pip-version-check --break-system-packages
     -r ${USERVER_ROOT_DIR}/scripts/external_deps/requirements.txt
   WORKING_DIRECTORY ${USERVER_ROOT_DIR}
 )
diff --git a/cmake/GrpcTargets.cmake b/cmake/GrpcTargets.cmake
index b644d46fb..3007ec1cd 100644
--- a/cmake/GrpcTargets.cmake
+++ b/cmake/GrpcTargets.cmake
@@ -45,7 +45,7 @@ if(NOT USERVER_IMPL_GRPC_REQUIREMENTS_CHECKED)
   endif()
   execute_process(
     COMMAND "${PYTHON}"
-      -m pip install --disable-pip-version-check
+      -m pip install --disable-pip-version-check --break-system-packages
       -r "${USERVER_DIR}/scripts/grpc/${file_requirements_protobuf}"
     RESULT_VARIABLE RESULT
     WORKING_DIRECTORY "${USERVER_DIR}"
diff --git a/external-deps/PostgreSQL.yaml b/external-deps/PostgreSQL.yaml
index 26ee0987c..4a4029ecc 100644
--- a/external-deps/PostgreSQL.yaml
+++ b/external-deps/PostgreSQL.yaml
@@ -15,7 +15,7 @@ includes:
 libraries:
     find:
       - names:
-          - libpq.a
+          - libpq.so
         paths:
           - ${USERVER_PG_LIBRARY_DIR}
 
